assets/css/style.css

```diff
@@ -1,111 +1,136 @@
-@media (min-width: 375px) {
+@media (min-width: 768px) {
   .container {
     display: flex;
     flex-direction: column;
     justify-content: center;
-    margin-left: 0;
-    margin-right: 0;
-    margin-top: 0;
-    margin-bottom: 0;
+    margin: 0;
+    background-image: url(../img/desk_bg.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
   }
 
   .header-menu {
     display: flex;
-    flex-direction: row-reverse;
-    margin-top: 5%;
-    height: 5%;
+    flex-direction: row;
+    justify-content: center;
+    height: 72px;
+    margin-top: 1%;
   }
 
   #hamburg-icon {
-    font-size: 17pt;
+    font-size: 17px;
     padding-right: 2rem;
     color: #28352f;
+    display: none;
   }
 
-  main {
-    background-image: url(../img/header-illsutration-mobile@2x.png);
-    background-size: 100%;
-    background-repeat: no-repeat;
-    margin-top: 5%;
+  nav ul {
+    display: flex;
+    justify-content: center;
+    height: 40px;
+    gap: 25px;
+  }
+
+  nav ul li {
+    list-style: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
   }
 
+  main { margin-top: 5%; }
+
   .socialmedia-links {
     display: flex;
     flex-direction: row;
-    margin-top: 6%;
     justify-content: center;
-    transform: translateX(-7%);
-    gap: 1.5%;
+    align-items: center;
+    margin-left: -2.5%;
   }
 
   .socialmedia-links li {
     list-style: none;
-    padding-left: 4%;
+    color: #505f79;
+    padding: 8px;
+    width: 40px;
+    height: 40px;
   }
 
-  .socialmedia-links img {
-    height: 90%;
-    width: 90%;
+  .headline-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-left: 18%;
+    margin-right: 18%;
+    margin-top: 5%;
```

Comment:
- [ ] Good job with the headline section 👍👍. However, from `768px` onwards, I noticed an empty space on right side which is not consistent with the design guidelines 😓 Kindly make sure that the desktop version should be fully responsive. Please fix these issues ✨ 

https://github.com/Graciano1997/portofolio-graciano/assets/106119105/703bf751-db6d-4ce3-a355-69e1858b2e00